adding klusterlet addon known issue (#4693)
open-cluster-management/rhacm-docs: release_notes/known_issues.adoc

The cluster claim is re-created automatically.

[#klusterlet-addon-search-crashing]
=== _klusterlet-addon-search_ pod fails
//2.5:27173

> The `klusterlet-addon-search` pod fails because the memory limit is reached. You must update the memory request and limit by customizing the `klusterlet-addon-search` deployment on your managed cluster. Edit the `ManagedclusterAddon` custom resource named `search-collector`, on your hub cluster. For example , if you have a managed cluster named `foobar`, run the following command:
> 
> 
> Add the following annotations to the `search-collector` custom resource to change the memory request and limit. In the following example, the request is set to `512Mi` and limit is set to `1024Mi`:

Reviewer: @dockerymick After my last edits, this didn't flow correctly.  Hope this makes it clear.

```suggestion
The `klusterlet-addon-search` pod fails because the memory limit is reached. You must update the memory request and limit by customizing the `klusterlet-addon-search` deployment on your managed cluster. Edit the `ManagedclusterAddon` custom resource named `search-collector`, on your hub cluster. Add these annotations and update the memory as needed.
* addon.open-cluster-management.io/search_memory_request=512Mi
* addon.open-cluster-management.io/search_memory_limit=1024Mi

For example, if you have a managed cluster named `foobar`, run the following command to change the memory  request to `512Mi` and the memory limit to `1024Mi`:
```

Author: Thanks for clarifying! Updated

The cluster claim is re-created automatically.

[#klusterlet-addon-search-crashing]
=== _klusterlet-addon-search_ pod fails
//2.5:27173

> The `klusterlet-addon-search` pod fails. You must update the memory request and limit by customizing the `klusterlet-addon-search` deployment on your managed cluster. Edit the `ManagedclusterAddon` custom resource named `search-collector`. For example , if you have a managed cluster named `foobar`, run the following command:

Reviewer: ```suggestion
The `klusterlet-addon-search` pod fails due to out-of-memory (OOM) error. You must update the memory request and limit by customizing the `klusterlet-addon-search` deployment on your managed cluster. On the hub cluster, edit the `ManagedclusterAddon` custom resource named `search-collector`. For example , if you have a managed cluster named `foobar`, run the following command:
```

> ----
> oc edit managedclusteraddon -n foobar search-collector
> ----
> 
> Add the following annotation to the `search-collector` custom resource to change the memory request and limit. View the following example, where the request is set to `512Mi` and limit is set to `1024Mi`:
> 
> ----
> kind: ManagedClusterAddOn
> metadata:
>   annotations:
>     addon.open-cluster-management.io/search_memory_limit: 1024Mi
>     addon.open-cluster-management.io/search_memory_request: 512Mi
> ----

Reviewer: The example can be simplified with a single command.

```suggestion

Add the following annotations to the `search-collector` custom resource to change the memory request and limit. In the following example, the request is set to `512Mi` and limit is set to `1024Mi`:

----
oc annotate managedclusteraddon search-collector -n foobar \
addon.open-cluster-management.io/search_memory_request=512Mi \
addon.open-cluster-management.io/search_memory_limit=1024Mi
----
```
